_posts/ShyArmadillo8/2016-02-04-JasminePlottChapter5Exercises.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+author: ShyArmadillo8
+title: "Jasmine Plott's Chapter 5 Exercises"
+---
+
+Here is the link to my first exercise:
+
+<iframe src="https://trinket.io/embed/python/8a5383a746" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+Here is the link to my second exercise:
+
+<iframe src="https://trinket.io/embed/python/4d24fbae0f" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+The first exercise wasn't so bad, but I had a really hard time with the second exercise and was unable to completely get it right in the end.  I have
+it set up so that it identifies the first digit of a number that you put in, but if you put in more than one number, even if the number 
+starts with a 9 but is smaller, the program spits out that this number is the maximum.  I tried changing the numbas variable I created to
```

Comment:
`numbas` is a string, so when you sort it you're alphabetizing it.  Also `for i in numbas:` will iterate through each character.

This is tricky and will become more clear with the weekend's reading.

I'd suggest converting the `numbas` datatype once at the beginning- that will probably solve it.